packages/react-dom/src/__tests__/ReactDOMForm-test.js

```diff
@@ -1670,6 +1670,48 @@ describe('ReactDOMForm', () => {
     expect(divRef.current.textContent).toEqual('Current username: acdlite');
   });
 
+  it('multiple form submissions in rapid succession do not throw', async () => {
+    const submitFormTwiceButtonRef = React.createRef();
+    let actionCounter = 0;
+    function App() {
+      const formRef = React.createRef();
+      // Submits the form twice in quick succession
+      // This simulates a user clicking a button twice in rapid succession
+      async function submitFormTwice() {
```

Comment:
I'm not sure if there is a better way to get two form actions to run independently